### DISCUSSION TITLE
Pip install

### DISCUSSION THREAD

**Discussion by jakeTPR0422** [iryna-kondr/scikit-llm#29] [Q&A] [answered]
Wondered why a simple pip install scikit-llm isn't working in Python3. Is there a certain dependency needed to begin with?

**Comment by OKUA1** [comment]
@jakeTPR0422 could you provide more details what exactly is not working / exact error message. Regular pop install command should be working

**Reply by jakeTPR0422** [reply]
I am getting the usual:

ERROR: could not find a version that satisfies the requirement scikit-llm (from versions: none)

**Reply by OKUA1** [reply]
I just tried to install scikit-llm in Google colab, everything seems to be working.

**Reply by OKUA1** [reply]
Please try to update your pip version first. pip install --upgrade pip

**Reply by jakeTPR0422** [reply]
Done - already have the latest version. I am doing this in the terminal via anaconda3

**Reply by OKUA1** [reply]
To be honest I am not sure what could be the problem. Which Python version are you using?

**Reply by jakeTPR0422** [reply]
3.6.8 - so I am doing an update now as assume it's a version issue

**Reply by OKUA1** [reply, marked as answer]
Yes. Python >= 3.8 is required.

**Reply by jakeTPR0422** [reply]
Thanks for your help.

**Reply by OKUA1** [reply]
No problem!

**Reply by badooori222** [reply]
Thanks for solution
